JoinColony/colonyCDapp
# Fix: remove most popular actions section

## Description

This PR removes the most popular actions section in the action sidebar as it will be getting replaced by another feature soon. The PopularAction component has also been removed as it's not used anywhere else within the app.

## Testing

1. Open action sidebar
2. Check if the most popular actions section is showing

## Diffs

**Changes** 🏗
Removed popular actions section:
![Screenshot 2024-02-13 at 12 43 42](https://github.com/JoinColony/colonyCDapp/assets/155957480/54b66dbd-3985-419e-914c-60d5c15d644e)

**Deletions** ⚰️

PopularActions.tsx

Resolves #1847
